### PR TITLE
Refactoring Project Config Files and Folders

### PR DESCRIPTION
**Pull request by em-torres**
* Changing project to Python 3
* Adding Python Decoupling
* Modifying Settings.py Credentials and Folder Config
* Changing Project Folders (Templates, Statics, Src)
* Updating and Testing Personal Environment

### CODE REVIEW

File: src/.env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+DEBUG=True
```
Comment (oldani) [inline src/.env]: This file should not be track, add this to the gitignore and also move to the mian dir ,instead you should add a .env.example with the vars values empty or a brief description to the repo

File: src/settings.py [hunk 2]
```diff
@@ -10,30 +10,33 @@
...
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'njb_o-$^^zt36b7==n1om1*_o2uv9h3pug$q6*c8powy#-+4of'
+SECRET_KEY = config('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = config('DEBUG', default=False, cast=bool)
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())
```
Comment (oldani) [inline src/settings.py:31]: This return a list?

File: templates/homepage/homepage_body.html [hunk 3]
```diff
@@ -313,5 +313,5 @@ <h3 id="nosotrosh3" class="nosotrosh3">Visión:</h3>
 {% endblock %}
 
 {% block custom_js %}
-	<script async src="{% static 'homepage/js/homepage.js' %}"></script>
+	<script async src="{% static 'homepage/homepage.js' %}"></script>
 {% endblock %}
```
Comment (oldani) [inline templates/homepage/homepage_body.html]: End of line missing here

File: templates/homepage/homepage_body.html [hunk 4]
```diff
@@ -313,5 +313,5 @@ <h3 id="nosotrosh3" class="nosotrosh3">Visión:</h3>
...
 
 {% block custom_js %}
-	<script async src="{% static 'homepage/js/homepage.js' %}"></script>
+	<script async src="{% static 'homepage/homepage.js' %}"></script>
```
Comment (oldani) [inline templates/homepage/homepage_body.html]: Just use `async` for js that do not depends on other js file, for example is this js file use jquery but as it's loading asyn i will load first than jQuery and not gonna find it and it's gonna break

File: templates/common/control_header.html [hunk 5]
```diff
@@ -13,10 +13,10 @@
 		<title>(Puerto Plata Express - {% block title %}{% endblock %})</title>
```
Comment (oldani) [inline templates/common/control_header.html]: Why two headers files with almost the same structure, templates inheritance is for avoid this.

File: create_env.sh [hunk 6]
```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
-echo "***** Creating and Activating Virtual Environment *****"
-virtualenv venv
+echo "\n***** Installing *****"
+echo "/* Installing Python, Pip and VirtualEnv */"
+sudo apt-get update
+sudo apt-get install python3-pip python3-dev build-essential
+pip install virtualenv virtualenvwrapper
```
Comment (oldani) [inline create_env.sh]: Why install virtualenv wrraper if not using it?

**Review by oldani — COMMENTED** [review]
Left some questions and advices. Also a will suggest using a task manager or bundle to package all the css files at least, cause it's not good for performance to have many request files

File: src/settings.py (hunk 2 above)
Comment (em-torres) [reply src/settings.py:31]: It will...

**Comment by oldani** [comment]
LGTM!

**Pull request merged**